emails/views.py

```diff
@@ -280,13 +289,21 @@ def _sns_message(message_json):
             )
             incr_if_enabled('email_for_deleted_address', 1)
             # TODO: create a hard bounce receipt rule in SES
+            return HttpResponse("Address deleted.", status=404)
```

Comment:
Assuming the user's in good standing (not abusing the system) then I think that:
- if the user is attempting to email to a defunct email address, we send them an email response saying address not available - an email 404 if you will.
- if it's not the user of a defunct email address, then we just eat the email quietly - so as to not alert the people that they've been spam blocked.

this will be based on from header which of course people could fake, but i think that's an acceptable edge case.

Reply:
Yeah let's make sure we consider those cases. In this particular code, these http responses are sent back to Amazon SNS, so I don't think they get any further than that. But maybe SNS could do something different based on the status code?